Resources/ServerInfo/Guidebook/Engineering/MixingAndFiltering.xml

```diff
@@ -0,0 +1,49 @@
+<Document>
+  # Mixing and Filtering
+  Gas mixers and filters are essential tools for manipulating the composition of gases within a pipe network.
+  <Box>
+    <GuideEntityEmbed Entity="GasMixer"/>
+    <GuideEntityEmbed Entity="GasFilter"/>
+  </Box>
+
+  Gas mixers and filters require power from a nearby LV cable to operate.
+
+  ## Gas Mixer
+  Gas mixers are used to combine gases in specific ratios within a pipe network. They are essential for creating controlled gas mixtures for various applications.
+
+  Gas mixers have 3 connections: 2 inputs and 1 output, as shown below:
+  <Box>
+    <GuideEntityEmbed Entity="GasPipeStraight" Caption="Primary" Rotation="90"/>
+    <GuideEntityEmbed Entity="GasMixer" Caption="" Rotation="90"/>
+    <GuideEntityEmbed Entity="GasPipeStraight" Caption="Output" Rotation="90"/>
+  </Box>
+<Box>
+  <GuideEntityEmbed Entity="GasPipeStraight" Caption="Side"/>
+</Box>
+
+  Gas mixers will still respect the requested gas mixture even if one of the input gases is not available. For example:
+  - If the requested mixture is 22% oxygen and 78% nitrogen, but there is no available oxygen, the mixer will not work until oxygen is available.
+  - If oxygen is available, but at a pressure lower than required to create the proper mixture at the requested pressure, the mixer will create the mixture, but the output will be lower pressure than requested.
```

Comment:
I think I read your review right. I wrote this:

Gas mixers also mix gasses based on pressure, not on mols. This can cause problems if the gasses are at different temperatures. For example:
- Presume a gas mixer was asked to mix Oxygen and Nitrogen at a ratio of 1:2 at a certain pressure.
- The Nitrogen in this case is double the temperature of the Oxygen.
- Hotter gas has more pressure, and thus fewer mols per volume than Oxygen.
- Because of this, Nitrogen will have half the mols compared to Oxygen, and the output will be 1:1 instead of 1:2.